src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -309,7 +309,7 @@ describe('RadioButtonGroup', () => {
 
           describe('when down arrow key is pressed twice', () => {
             describe('on one down arrow key press', () => {
-              it('should change the focus to the last radio button', () => {
+              it('should change the focus to the second radio button', () => {
```

Comment:
there are 3 radio buttons, [0, 1, 2]. radio button with key 2 is (3rd radio button aka last)

Reply:
I didn't change them intentionally, I must have weird state locally, but all I did was pull origin and update the function above